winui/AvatarView/content-types.md

```diff
@@ -1,10 +1,267 @@
 ---
 layout: post 
 title: Content Types in WinUI AvatarView control | Syncfusion
-description: Learn what are the content types available and how to set value for the content types in WinUI AvatarView control.
+description: Learn about the available content types and how to set values for them in the WinUI AvatarView control.
 platform: WinUI
 control: AvatarView
 documentation: ug
 ---
 
-# Content Types in WinUI AvatarView
\ No newline at end of file
+# Content Types in WinUI AvatarView
+
+The AvatarView control supports five different types of content: `Default`, `Initials`, `CustomImage`, `AvatarCharacter`, and `GroupView`.
+
+## Default
+
+Default content type is used for displaying the `Avatar1` character image when initializing without the initials, custom image, or group view types.
+
+{% tabs %}
+{% highlight xaml %}
+
+<Grid>
+<syncfusion:SfAvatarView />
+</Grid>
+
+{% endhighlight %} 
+{% highlight C# %}
+
+// Creating an instance of the AvatarView control.
+SfAvatarView avatarView = new SfAvatarView();
+           
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control](avatarview_images/winui_avatarview.png)
+
+## Initials 
+
+Initials content type is used to show initials to the `AvatarView`. When using the `ContentType` as initials, you need to set the initial character using the following properties:
+
```

Comment:
@MadhumithanJayapathi The initials content type is used to display initials in the `AvatarView`. To use this content type, you must provide a value for the `AvatarName` property, which sets the initials to be displayed in the avatar view. Depending on the specified `InitialsType` property, the initials will be formatted accordingly.